lib/maps.rb

```diff
@@ -20,8 +20,9 @@ def place_to_geodata_point(segment, storyline_timezone)
      'endtime' => Time.parse(segment['endTime']).to_i}
   end
 
-  def trackpoints_to_geodata_multiline(activity, trackpoints)
+  def move_to_geodata_multiline(move)
```

Comment:
Assignment Branch Condition size for move_to_geodata_multiline is too high. [19.67/15]